pki.spec

```diff
@@ -814,6 +814,13 @@ This package contains PKI test suite.
 %build
 ################################################################################
 
+# get Java <major>.<minor> version number
+java_version=`%{java_home}/bin/java -XshowSettings:properties -version 2>&1 | sed -n 's/ *java.version *= *\([0-9]\+\.[0-9]\+\).*/\1/p'`
```

Comment:
(nitpick) I understand that this is a backport of a working version of code. But, was curious on why we didn't use regex over `java -version`?

Reply:
It is a regex over `java -version` with the additional flag of `-XshowSettings:properties` :-) The latter makes the output more parseable apparently.

Reply:
`java -version` does show the Java version in a human readable format, but there's no guarantee that it will be consistent across versions or other vendors that we might support in the future, so it's safer to parse the `java.version` property specifically instead.